gcp/api/server.py

```diff
@@ -66,6 +66,8 @@
 # This should match the number in the indexer
 _BUCKET_SIZE = 512
 
+MAX_PACKAGE_NAME_LENGTH = 100  # Maximum allowed package name length
```

Comment:
Based on the error message cited in #1452 I think 1500 is the 

```suggestion
MAX_PACKAGE_NAME_LENGTH = 1500  # Maximum allowed package name length
```